library/std/src/sys/vxworks/ext/fs.rs

```diff
@@ -655,7 +652,7 @@ impl MetadataExt for fs::Metadata {
 /// Adds support for special Unix file types such as block/character devices,
 /// pipes, and sockets.
 ///
-/// [`FileType`]: ../../../../std/fs/struct.FileType.html
+/// [`FileType`]: fs::FileType
```

Comment:
I think it may be better to just use `fs::FileType` above.

Reply:
you mean like this

```rust
/// Unix-specific extensions for [`fs::FileType`].

```

Reply:
Yes